assets/xml/overlays/ovl_Obj_Jgame_Light.xml

```diff
@@ -1,12 +1,11 @@
-
 <Root>
     <File Name="ovl_Obj_Jgame_Light" BaseAddress="0x80C152F0" RangeStart="0x8FC" RangeEnd="0xE40">
         <!-- @bug: Based on the DList, this should be an invalid i16 -->
-        <Texture Name="sObjJgameLightIncorrectTex" OutName="obj_jgamelight_incorrect" Static="On" Format="ia16" Width="16" Height="16" Offset="0x8FC"/>
+        <Texture Name="sObjJgameLightIncorrectTex" OutName="obj_jgamelight_incorrect" Format="ia16" Width="16" Height="16" Offset="0x8FC"/>
```

Comment:
What did `Static="On"` do?

Reply:
It tells ZAPD to use `static` on the extracted asset.
I removed it while testing some stuff and forgot to put it back.
Just checked the rest of the codebase and we don't really use it so I guess we could just remove it for consistency?

Reply:
Yeah, I saw that this file was the only one that used it so I was wondering